topics/single-cell/tutorials/bulk-deconvolution-evaluate/tutorial.md

```diff
@@ -0,0 +1,573 @@
+---
+layout: tutorial_hands_on
+
+title: Evaluating Reference Data for Bulk RNA Deconvolution
+subtopic: deconvo
+priority: 3
+zenodo_link: 'https://zenodo.org/records/5719228'
+
+questions:
+- How do I evaluate my reference data?
+- How do I compare different deconvolution tools?
+- What are the best metrics for determining tool accuracy?
+objectives:
+- Generate psuedo-bulk data from single-cell RNA data
+- Process the single-cell and psuedo-bulk data using various deconvolution tools
+- Evaluate and visualse the results of the different deconvolution methods
+time_estimation: 2H
+key_points:
+- It is important to validate the accuracy of both deconvolution tools and reference data
+- There are various visualisation and quantative methods of analysing results
+- Different deconvolution tools have varying accuracy, comparing them against the same reference is a useful test to determine the best tool for your data
+
+tags:
+- transcriptomics
+
+contributions:
+  authorship:
+    - hexhowells
+  funding:
+    - elixir-fair-data

```

Comment:
Thanks for taking the time to review this @carloscheemendonca! We now also support crediting reviewers on tutorials, so please go ahead and add yourself to our CONTRIBUTORS.yaml file and edit this contribution metadata as follows:

```suggestion
contributions:
  authorship:
    - hexhowells
  reviewing:
    - carloscheemendonca
  funding:
    - elixir-fair-data
```